### ISSUE TITLE
Complete order button loading endless

### ISSUE THREAD

**Issue by Kuhno92** [harrowmykel/flutter_paypal_native#20]
<img width="383" alt="image" src="https://github.com/harrowmykel/flutter_paypal_native/assets/5812823/7b8e5aa1-1311-4dc0-a679-a4bb29eb5f50">

Any idea why the blue confirmation button has this endless loading animation?

**Comment by Philnue** [comment]
Hey, @Kuhno92 

Do you see in the Sandbox mode the payer and cart data ? 
For me it is empty.

**Comment by harrowmykel** [comment]
I dont know why it does the the endless loop but you can click it twice and it works

**Comment by Kuhno92** [comment]
@Philnue yes I can see everything in sandbox mode. Only the endless loop is always there

**Comment by harrowmykel** [comment]
Unfortunately this seem to be a problem from paypal :(

**Comment by harrowmykel** [comment]
Please upgrade to new version v2.0.0 . You have to remove something from android manifest. See readme.md

**Comment by waqas-alii** [comment]
@harrowmykel not working in v2.0.0 , still getting the same endless loop. Any update or workaround for this?

**Comment by harrowmykel** [comment]
Try Version 2.0.2

**Comment by waqas-alii** [comment]
Did that but still got the same endless loader

**Comment by harrowmykel** [comment]
Just click the button again. It should work. The problem is from paypal unfortunately

**Comment by waqas-alii** [comment]
It works fine on Android but on iOS it's the same endless loader.

**Comment by alperragib** [comment]
I have the same problem and it continues. How do we solve this? Can you help?

**Comment by Navarad** [comment]
The problem persists on iOS and package version 2.1.0

**Comment by ot-repo** [comment]
Hello, any update on this issue? This confuses the customer as they think it's still loading and do not press on the button without a text. Without fixing this problem we cannot use the plug in. Hope you can deliver a fast solution.

**Comment by harrowmykel** [comment]
Unfortunately i dont know how to solve the issue. It is from the ios paypal package :(

**Comment by ot-repo** [comment]
This is not right, I have just run the demo application given with the ios sdk and it worked as it should be without spinning icon. Maybe they have fixed it?

https://github.com/paypal/paypal-ios

**Comment by harrowmykel** [comment]
@ot-repo can you upload your code on github so that i can compare the implementation?

**Comment by ot-repo** [comment]
@harrowmykel it's already on the same repository:

https://github.com/paypal/paypal-ios/tree/main/Demo/Demo

The native checkout is here:

https://github.com/paypal/paypal-ios/blob/main/Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift

**Comment by ot-repo** [comment]
@harrowmykel In the read.me file it's documented how you can start the demo application:

Demo
Open PayPal.xcworkspace in Xcode
Resolve the Swift Package Manager packages if needed: File > Packages > Resolve Package Versions or by running swift package resolve in Terminal
Select the Demo scheme, and then run

**Comment by ot-repo** [comment]
Hi @harrowmykel  any update on this issue?

**Comment by ot-repo** [comment]
Hi @harrowmykel I am running out of time. I need to somehow solve this problem. Could you compare your implementation with the demo application? anything I can help you with? I appreciate your feedback.

**Comment by harrowmykel** [comment]
Hey I am so sorry to reply now. This package uses paypal checkout sdk which is deprecated by PayPal. 
The demo app that you sent uses the new PayPal ios sdk which is different.
I have not been able to fix the problem and I do not think that I can fix it. I may have to move to the new sdk but I unfortunately currently do not have the time for it at the moment.

**Comment by ot-repo** [comment]
Alright, thank you anyway!

**Comment by harrowmykel** [comment]
Yes sure. How would you like to contact me?
You can use Contactpiccmaq@gmail.com if you would like to send an email

**Comment by Rishan99** [comment]
Any Progress on this issue?

**Comment by keyurmavadiya** [comment]
Any one found the solution?

**Comment by sammrafi** [comment]
I created a new plugin with an updated SDK and some improvements. This problem has been fixed in this plugin.

Check out this plugin: [PayPal Native Checkout](https://pub.dev/packages/paypal_native_checkout).
